src/rpc/net.cpp

```diff
@@ -203,7 +205,10 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
         // their ver message.
         obj.pushKV("subver", stats.cleanSubVer);
         obj.pushKV("inbound", stats.fInbound);
-        obj.pushKV("addnode", stats.m_manual_connection);
+        if (IsDeprecatedRPCEnabled("getpeerinfo_addnode")) {
```

Comment:
let's:
  - drop this `if`
  - drop label "it is deprecated" from rpc help
  - drop `rpc_getpeerinfo_deprecation.py`

 And split this PR to 2 PR.
One PR (this PR) will get #19725 as partial (and not actually deprecating old field); but new PR (with trivial changeset) will get merged in v21

@UdjinM6 ?

Reply:
Yes, we can go this way I guess